tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -268,10 +282,8 @@ initialize_IotResponseHandle(IotHttpsResponseHandle_t pResponseHandle) {
 }
 
 int is_valid_IotResponseHandle(IotHttpsResponseHandle_t pResponseHandle) {
-  int valid_headers =
-    pResponseHandle->pHeaders != NULL;
-  int valid_body =
-    pResponseHandle->pBody != NULL;
+  int valid_headers = pResponseHandle->pHeaders != NULL;
```

Comment:
Why change line breaks?  This was formatted so that all predicate definitions were similar.

Reply:
But it's not consistent. For example, lines 358 and 359 do not use line breaks.

https://github.com/aws/amazon-freertos/pull/2408/files/fde2d074464d339da519ef27d5760ecc47acf20b#diff-a5a8d6b34fac01a0b4afe4254cb313d2R358-R359

Reply:
I can change both functions to whichever style the team uses, but it should be consistent.